vdb/lib/osv.py

```diff
@@ -198,7 +202,15 @@ def to_vuln(cve_data):
                             score = cvss.get("score")
                             severity = convert_score_severity(score)
             user_interaction = "REQUIRED"
-            if vector_string:
+            # We give importance to cvss4 which always seems to be lower compared to cvss3
+            if cvss4_vector_string:
```

Comment:
@prabhu Can we not include both v3 and v4 scores since metrics is an array? Omitting v3 scores when v4 are available will mean we are unable to include any scoring for CSAF.

Reply:
Both must be included. Only the preference for the default severity and score has changed to cvss v4 values since they are always lower compared to cvss v3

Reply:
ah didn't see the elif.